### PR TITLE
Move impeller-cmake-example to flutter/third_party.

### PR DESCRIPTION
**Pull request by matanlurey**
... as part of buildmoot.

I _think_ this is all that is needed but am not confident.

### CODE REVIEW

**Review by bdero — APPROVED** [review]
LGTM!

File: DEPS [hunk 1]
```diff
@@ -1008,7 +1008,7 @@ deps = {
      'dep_type': 'cipd',
    },
 
-  'src/third_party/impeller-cmake-example': {
+  'src/flutter/third_party/impeller-cmake-example': {
```
Comment (bdero) [inline DEPS:1011]: I think one more change is needed for this actually

Comment (bdero) [reply DEPS:1011]: I think this path needs to be updated: https://github.com/flutter/engine/blob/e03cf86c4170f9c010c5189aa6299609c41b88d3/DEPS#L1139

Comment (matanlurey) [reply DEPS:1011]: Ah good catch, ack

**Pull request merged**